resources/views/backend/layouts/app.blade.php

```diff
@@ -15,6 +15,7 @@
           rel="stylesheet">
 
     <link href="{{ asset('css/backend/app.css') }}" rel="stylesheet">
+    <link href="{{ asset('css/backend/toastr.css') }}" rel="stylesheet">
```

Comment:
Toastr wasn't updated since on 16 Jun 2018, bit outdated...

Reply:
Aha, but i only needed it to serve the case instead of creating many forms and modal..etc

Reply:
You can create a blade view as a skeleton and include it everywhere you want for example. Using libarays for not just creating a "many" forms and modals is just lazy. Guess I'll create a `modal.blade.php` as a skeleton to include it everywhere I needed.

Reply:
Isn't using toastr with axios faster ? 🤔 Because i will need it a lot in the future in the whooe backend because of the new datatables i am working on.